File: pint.m
#include "monty.h"

/**
 * pint - Prints top value.
 * @stack: Point top of the stack.
 * @line_numm: Line number in the file where the opcode is located.
 */
void pint(stack_t **stack, unsigned int line_numm)
{
    stack_t *top;

    if (*stack == NULL)
    {
        fprintf(stderr, "L%u: can't pint, stack empty\n", line_numm);
        exit(EXIT_FAILURE);
    }

    top = *stack;
    printf("%d\n", top->n);
}

File: pint.c
#include "monty.h"

/**
 * f_pint - pr!nts_t0p
 * @head: st@ck h£@d
 * @counter: l!n£_num
 * Return: n0th!ng
 */
void f_pint(stack_t **head, unsigned int counter)
{
	if (*head == NULL)
	{
		fprintf(stderr, "L%u: can't pint, stack empty\n", counter);
		fclose(bus.file);
		free(bus.content);
		free_stack(*head);
		exit(EXIT_FAILURE);
	}
	printf("%d\n", (*head)->n);
}

